src/p_mobj.c

```diff
@@ -58,6 +58,16 @@ void P_UpdateDirectVerticalAiming(void)
   max_pitch_angle = default_max_pitch_angle * ANG1;
 }
 
+void P_SetActualHeight(mobj_t *mobj)
+{
+    spritedef_t *sprdef = &sprites[mobj->sprite];
+    spriteframe_t *sprframe = &sprdef->spriteframes[mobj->frame & FF_FRAMEMASK];
+    if (sprframe)
```

Comment:
We don't need to consider an `else` case, right?

Reply:
Hmmm, should we set it to `0` in the `else` case?

Reply:
Or `mobj->height >> FRACBITS`.